projects/spectator/test/set-input/set-input.component.spec.ts

```diff
@@ -6,52 +6,83 @@ import { SetInputComponent } from './set-input.component';
 describe('SetInputComponent', () => {
   let spectator: Spectator<SetInputComponent>;
 
-  const createComponent = createComponentFactory({
-    component: SetInputComponent,
-    imports: [CommonModule]
-  });
+  describe('with default enabled detectChanges', () => {
+    const createComponent = createComponentFactory({
+      component: SetInputComponent,
+      imports: [CommonModule]
+    });
 
-  beforeEach(() => {
-    spectator = createComponent();
-  });
+    beforeEach(() => {
+      spectator = createComponent();
+    });
 
-  it('should be update input from object', () => {
-    spectator.setInput({ one: '1' });
+    it('should be set value from OnInit angular hook', () => {
+      expect(spectator.component.fromInit).toBe('initValue');
+    });
 
```

Comment:
I'm not sure what these changes are.

Reply:
I want to test 2 cases. I added 2 describes:
- defaults case (detectChanges = true)
- detectChanges = false